<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UserWindow</class>
 <widget class="QMainWindow" name="UserWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1418</width>
    <height>884</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>1418</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>UserWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableWidget" name="userCollectionTableWidget">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>40</y>
      <width>591</width>
      <height>381</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Title</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Author</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Review</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="recommendedCollectionTableWidget">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>510</y>
      <width>591</width>
      <height>261</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Title</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Author</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Review</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="collectionLabel">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Collection</string>
    </property>
   </widget>
   <widget class="QLabel" name="recommendationsLabel">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>480</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Recommendations</string>
    </property>
   </widget>
   <widget class="QWidget" name="addBookWidget" native="true">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>3</y>
      <width>280</width>
      <height>821</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="titleLabel">
       <property name="text">
        <string>Title</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="titleLineEdit">
       <property name="text">
        <string>Foundation</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="authorLabel">
       <property name="text">
        <string>Author</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="authorLineEdit">
       <property name="text">
        <string>Isaac Asimov</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="reviewLabel">
       <property name="text">
        <string>Review</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QTextEdit" name="reviewTextEdit">
       <property name="html">
        <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;meta charset=&quot;utf-8&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
hr { height: 1px; border-width: 0; }
li.unchecked::marker { content: &quot;\2610&quot;; }
li.checked::marker { content: &quot;\2612&quot;; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;First SF book to capture my imagination, you should read the robots series first as they set the scene. Yet to find the series in Kindle!&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="addBookPushButton">
       <property name="text">
        <string>Add Book</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="saveCollectionPushButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>430</y>
      <width>291</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Collection</string>
    </property>
   </widget>
   <widget class="QPushButton" name="loadCollectionPushButton">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>430</y>
      <width>291</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Load Collection</string>
    </property>
   </widget>
   <widget class="QPushButton" name="recommendPushButton">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>780</y>
      <width>591</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Recommend</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="supplierCollectionTableWidget">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>40</y>
      <width>451</width>
      <height>731</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Title</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Author</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Review</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="catalogLabel">
    <property name="geometry">
     <rect>
      <x>950</x>
      <y>20</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Catalog</string>
    </property>
   </widget>
   <zorder>addBookWidget</zorder>
   <zorder>userCollectionTableWidget</zorder>
   <zorder>recommendedCollectionTableWidget</zorder>
   <zorder>collectionLabel</zorder>
   <zorder>recommendationsLabel</zorder>
   <zorder>saveCollectionPushButton</zorder>
   <zorder>loadCollectionPushButton</zorder>
   <zorder>recommendPushButton</zorder>
   <zorder>supplierCollectionTableWidget</zorder>
   <zorder>catalogLabel</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1418</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuUser_View">
    <property name="title">
     <string>User View</string>
    </property>
   </widget>
   <addaction name="menuUser_View"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
